.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Bug Report
+about: Create a report to help us improve
+
+---
+
+<!--
+YOUR ISSUE MAY BE CLOSED IF YOU DO NOT FOLLOW THIS TEMPLATE
+
+Consider searching for similar issues before submitting yours.
+-->
+
+**To Reproduce:** <!-- A simple set of reproduction steps or a MINIMAL reproducible example -->
+
+**Expected Behavior:** <!-- What you expect to happen in the context of the bug. -->
+
+**Actual Behavior:** <!-- What actually happens instead. If an error occurs, include the stacktrace here. -->
+
+**Version:** <!-- The version of Discord4J on which the bug is produced. -->
```

Comment:
Maybe an additional section for misc additional info?

Reply:
Im not sure what other information is relevant. I found this section to almost always be empty or contain un-needed info in the current template.

Reply:
Well, the point is that we can't predict every case. There may be some additional context required or maybe a demo of the issue in action like images, etc.

Reply:
Would the "Actual Behavior" section not be better to display the issue in action?

Reply:
Maybe that was a poor example. What if there are some weird circumstances the user is not sure is causing the issue but wants us to be aware of? Like a patchy internet connection or a weird store setup.